user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr :hoge
+  def initialize
+    @hoge =    [a,b,c,d]*''
+    # にゃーん
+  end
```

Comment:
__[staging][rubocop]__
* Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)

<!-- Comment by [SideCI](https://sideci.com) -->